web-ng/src/app/components/map/map.component.ts

```diff
@@ -163,42 +138,109 @@ export class MapComponent implements OnInit, AfterViewInit, OnDestroy {
     }
   }
 
-  onFeatureClick(featureId: string | null) {
-    this.navigationService.setFeatureId(featureId);
+  private deleteMarkers() {
```

Comment:
It seems this is only ever called together with `clearGoogleMapDataLayer()`. Can they be combined into one method? The combined method name might then be something like `clearMap()` to make it clearer what this is doing.